fdbclient/FileBackupAgent.actor.cpp

```diff
@@ -698,10 +698,7 @@ struct EncryptedRangeFileWriter : public IRangeFileWriter {
 		copyToBuffer(self, s->begin(), s->size());
 	}
 
-	static bool isSystemKey(KeyRef key) { return key.size() && key[0] == systemKeys.begin[0]; }
-
-	ACTOR static Future<EncryptCipherDomainId> getEncryptionDomainDetailsImpl(KeyRef key,
```

Comment:
Thanks for updating it. Looking at the old implementation, am not sure if I understood why it was an ACTOR

Reply:
Earlier we were using the tenant cache which had futures. I guess we never came around to changing it until now